index.html

```diff
@@ -38,8 +37,152 @@ <h2 class="connect">LET'S CONNECT</h2>
                         <li class="social-icon"><a href="#"><img src="./image/icons/medium.svg" alt="medium"></a></li>
                         <li class="social-icon"><a href="#"><img src="./image/icons/github.svg" alt="github"></a></li> 
                         <li class="social-icon"><a href="#"><img src="./image/icons/angeli.svg" alt="angeli"></a></li>                 
+           
                     </ul>
         </section>
+        <section class="works-section">
+             <div class="project">
+                <div class="project-img">
+                    <img src="image/project-image.png" alt="Project image">
```

Comment:
- [ ] This image is well placed but to keep things simple, this is a background image. And should be set in the CSS rather than an `img` tag.

Reply:
I Don't understand the issue with this please advise

Reply:
the img is a project img hence it cannot be put in the CSS. i have made lots of inquiries and it looks like an honest mistake from the reviewer

Reply:
Hi @HanningtoneM, you are right. This was a mistake. Please proceed with the other suggestion.